hbase-shell/src/main/ruby/hbase/taskmonitor.rb

```diff
@@ -107,88 +107,5 @@ def tasksOnHost(filter, host)
 
       tasks
     end
-
-    #---------------------------------------------------------------------------------------------------
-    # Prints a table of filtered tasks on requested hosts
-    def tasks(filter, hosts)
```

Comment:
unused? Or you use your new formatter now?

Reply:
The new formatter is in use now and some of this logic moved to Processlist#_processlist_impl. This method is very tailored to printing actual output, which by convention goes in the command file itself. As it is not general purpose, I don't feel that this method is worth keeping around, but I can leave it and mark it as deprecated if you think that would be better.

Reply:
ok. Just drop it is good.